physionet-django/project/views.py

```diff
@@ -56,6 +56,12 @@
 from user.forms import AssociatedEmailChoiceForm
 from user.models import CloudInformation, CredentialApplication, LegacyCredential, User, Training
 
+from .serializers import PublishedProjectSerializer, PublishedProjectDetailSerializer
```

Comment:
Please could you use the absolute import, instead of relative?

Reply:
Changed it.